src/components/Home/Home.tsx

```diff
@@ -45,149 +44,142 @@ const Home = ({ personalities, stats, href, isLoggedIn }) => {
                                         </div>
                                     )
                             )}
                         </Carousel>
-                    </div>
+                    </Col>
                     <div
                         style={{
                             position: "absolute",
                             zIndex: 2,
                             width: "100%",
-                            padding: "7%"
+                            height: "100%",
                         }}
                     >
-                        <Row
-                            style={{
-                                height: "25%",
-                                color: "#fff",
-                                flexDirection: "column",
-                                textAlign: "center"
-                            }}
+                        <Col
+                            span={18}
+                            offset={3}
+                            className="header-container"
                         >
-                            <h1
-                                style={{
-                                    color: "#fff",
-                                    fontSize: "26px"
-                                }}
+                            <Col
+                                offset={4}
+                                span={16}
+                                className="header-content-container"
                             >
-                                {t("home:title")}
-                            </h1>
-                            <span>{t("home:subtitle")}</span>
-                        </Row>
-                        <Row
-                            style={{
-                                color: "#fff",
-                                flexDirection: "column"
-                            }}
-                        >
-                            <Row>
-                                <p>
-                                    <span
-                                        style={{
-                                            color: "#67BEF2",
-                                            fontSize: "34px"
-                                        }}
+                                <Row className="home-title-container">
+                                    <h1 className="home-title">
+                                        {t("home:title")}
+                                    </h1>
+                                    <h2 className="home-subtitle">
+                                        <span className="home-subtitle0">{t("home:subtitle0")}</span>{" "}
+                                        <span className="home-subtitle1">{t("home:subtitle1")}</span>
+                                    </h2>
+                                </Row>
+                                {!isLoggedIn &&
+                                    <Row className="footer-container">
```

Comment:
Calling it footer is confusing because that's not the page footer.

Let's move it to a new component and call it `CTASection`